parsec/api/data/manifest.py

```diff
@@ -120,6 +123,8 @@ def type_schemas(self):
         def get_obj_type(self, obj):
             return obj["type"]
 
+    version: int
```

Comment:
This should be ni the `SCHEMA_CLS` then

Reply:
fixed

Reply:
`author` should also be added...

Reply:
In fact that was a miss assumption, i thought that manifest are always created with an author, but in fact they are just always signed with an author (and the object can be setup with None like in parsec/api/data/manifest.py ) I roll this back